webapp/src/pages/createForm/components/Question.js

```diff
@@ -0,0 +1,265 @@
+    id: inputs.id,
+    order: inputs.order,
+    type: inputs.type,
+    options: inputs.options,
+    value: inputs.value,
+    label: inputs.label,
+    required: inputs.required
+  });
+
+  const handleChange = (prop) => (e) => {
+    setInput({...input, [prop]: e.target.value});
+  }
+  const handleTypeChange = (e) => {
+    setInput({...input, type: e});
+  }
+  const handleCheckChanged = (e) => {
+    setInput({...input, 'required': e.target.checked});
+  }
+
+  const handleAddOption = () => {
+    setInput({...input, options: [{
+      id: `${Math.random()}`,
+      value: input.value,
+      label: input.label,
+    }, ...input.options]});
+  }
+  const resetInputs = () => {
+    setInput({...input, value: '', label: ''});
+  }
+  const handleDeleteOption = (id, e) => {
+    const newOptions = input.options.filter(e => e.id !== id);
+    setInput({...input, options: [...newOptions]})
+  }
+
+  const updateQuestions = () => { // Update questions in the sections form when question loses focus
+    questions[num-1] = input;
+    setQuestions([...questions]);
+  }
+
+  const option = props => {
+    const { value } = props;
+    const label = <div>
+      <div className='dropdown-text'>
+        <i className={props.faClass}></i>
+      </div>
+      {t(props.label)}
+    </div>
+    return {
+      label,
+      value
+    }
+  }
+
+  const options = [
+    option({
+      value: 'short-text',
+      label: 'Short Text',
+      faClass: 'fa fa-align-left fa-xs fa-color'
+    }),
+    option({
+      value: 'long-text',
+      label: 'Long Text',
+      faClass: 'fas fa-align-justify fa-color'
+    }),
+    option({
+      value: 'markdown',
+      label: 'Markdown',
+      faClass: 'fab fa-markdown fa-color'
+    }),
+    option({
+      value: 'single-choice',
+      label: 'Single Choice',
+      faClass: 'fas fa-check-circle fa-color'
+    }),
+    option({
+      value: 'multi-choice',
+      label: 'Multi Choice',
+      faClass: 'far fa-caret-square-down fa-color'
+    }),
+    option({
+      value: 'multi-checkbox',
+      label: 'Multi Checkbox',
+      faClass: 'fas fa-check-square fa-color'
+    }),
+    option({
+      value: 'file',
+      label: 'File',
+      faClass: 'fas fa-cloud-upload-alt fa-color'
+    }),
+    option({
+      value: 'date',
+      label: 'Date',
+      faClass: 'fas fa-calendar-alt fa-color'
+    }),
+    option({
+      value: 'reference',
+      label: 'Reference',
+      faClass: 'fas fa-user fa-color'
+    }),
+    option({
+      value: 'multi-file',
+      label: 'Multi File',
+      faClass: 'fas fa-cloud fa-color'
+    }),
+  ];
+  const withPlaceHolder = ['short-text', 'multi-choice', 'long-text', 'markdown'];
+  const withOptions = ['multi-choice', 'multi-checkbox'];
+  return (
+    <>
+      <div
+        className="section-wrapper"
+        onBlur={updateQuestions}
+      >
+        <div className="headline-description">
+          <div className="question-header">
+            <input
```

Comment:
Please add a placeholder, it's not obvious at the moment that the user should type something here :)